package.json

```diff
@@ -108,8 +120,11 @@
   ],
   "dependencies": {
     "core-js": "^3.19.2",
+    "http-server": "^14.1.0",
```

Comment:
Being that we already have a simple web server, is something that we've added requiring this particular one?

Reply:
This was used before chromatic, and now that i am using chromatic I can remove it. Thanks